server/locales/en.json

```diff
@@ -329,9 +329,17 @@
   "admin.groups.importSuccess": "Rules imported successfully!",
   "admin.groups.info": "Group Info",
   "admin.groups.name": "Group Name",
+  "admin.groups.nameInvalidChars": "The group name contains invalid characters.",
   "admin.groups.nameHint": "Name of the group",
   "admin.groups.overview": "Overview",
   "admin.groups.permissions": "Permissions",
+  "admin.groups.permissions.adminHint": "Can access the administration area.",
+  "admin.groups.permissions.userHint": "Can create / manage users (but not users with administrative permissions)",
+  "admin.groups.permissions.groupHint": "Can create / manage groups and assign permissions (but not manage:system) / page rules",
+  "admin.groups.permissions.navigationHint": "Can manage site navigation",
+  "admin.groups.permissions.themeHint": "Can modify site theme settings",
+  "admin.groups.permissions.sitesHint": "Can create / manage sites",
+  "admin.groups.permissions.systemHint": "Can manage and access everything. Root administrator.",
```

Comment:
You can't add sub-keys to a key that already has a value.

Subsequent keys should instead use the format `admin.groups.permissionsAdminHint`